google-cloud-firestore/src/test/java/com/google/cloud/firestore/OpenTelemetryOptionsTest.java

```diff
@@ -95,5 +104,53 @@ public void canDisableTelemetryCollectionWhileOpenTelemetryInstanceIsNotNull() {
     assertThat(firestore.getOptions().getOpenTelemetryOptions().getEnabled()).isFalse();
     assertThat(firestore.getOptions().getOpenTelemetryOptions().getOpenTelemetry())
         .isEqualTo(openTelemetry);
+    assertThat(firestore.getOptions().getTraceUtil()).isNotNull();
+    assertThat(firestore.getOptions().getTraceUtil() instanceof DisabledTraceUtil).isTrue();
+  }
+
+  @Test
+  public void existenceOfGlobalOpenTelemetryDoesNotEnableTracing() {
+    // Register a global OpenTelemetry SDK.
+    OpenTelemetrySdk.builder().buildAndRegisterGlobal();
+
+    // Make sure Firestore does not use GlobalOpenTelemetry by default.
+    FirestoreOptions firestoreOptions = getBaseOptions().build();
+    firestore = firestoreOptions.getService();
+    assertThat(firestore.getOptions().getOpenTelemetryOptions().getEnabled()).isFalse();
+    assertThat(firestore.getOptions().getOpenTelemetryOptions().getOpenTelemetry()).isNull();
+    assertThat(firestore.getOptions().getTraceUtil()).isNotNull();
+    assertThat(firestore.getOptions().getTraceUtil() instanceof DisabledTraceUtil).isTrue();
+  }
+
+  @Test
+  public void canPassOpenTelemetrySdkInstanceToFirestore() {
+    OpenTelemetrySdk myOpenTelemetrySdk = OpenTelemetrySdk.builder().build();
+    FirestoreOptions firestoreOptions =
+        getBaseOptions()
+            .setOpenTelemetryOptions(
+                FirestoreOpenTelemetryOptions.newBuilder()
+                    .setTracingEnabled(true)
+                    .setOpenTelemetry(myOpenTelemetrySdk)
+                    .build())
+            .build();
+    firestore = firestoreOptions.getService();
+    EnabledTraceUtil enabledTraceUtil = (EnabledTraceUtil) firestore.getOptions().getTraceUtil();
```

Comment:
Add null check assertion for `firestore.getOptions().getTraceUtil()`?

Reply:
Done.